backend/src/models/Event.js

```diff
@@ -0,0 +1,40 @@
+const Table = require("./Table");
+
+class Event extends Table {
+  constructor(id) {
```

Comment:
Do you want to validate the info first the same way you validated the user info?

Reply:
That was not final solution, It works for now but it's inconvenient, requires each time realize validation in child class. What if we define 2 lists with required fields and optional fields and later will validate input data against these lists. All validation will be abstracted and put to `Table`, and in `User` we will define just these 2 lists and use generic validator.

Reply:
Sounds great to me!